devicetypes/iblinds/iblinds-zwave.src/iblinds-zwave.groovy

```diff
@@ -16,14 +16,13 @@ import groovy.json.JsonOutput
 
 metadata {
 	definition (name: "iblinds Z-Wave", namespace: "iblinds", author: "HABHomeIntel", ocfDeviceType: "oic.d.blind",  mnmn: "SmartThings", vid: "generic-shade-3") {
+		capability "Window Shade"
+		capability "Window Shade Preset"
 		capability "Switch Level"
-		capability "Switch"
```

Comment:
I removed this since this DTH isn't actually using `Switch`, and that isn't recommended for window covering devices any more.